internal/u64/u64.go

```diff
@@ -0,0 +1,15 @@
+package u64
+
+// LeBytes returns a byte array corresponding to the 8 bytes in the uint64 in little-endian byte order.
+func LeBytes(v uint64) []byte {
```

Comment:
is this exactly the same as `binary.LittleEndian.PutUint64`?

Reply:
yep, except with that you have to pre-allocate an array to put it into which makes the call sites suffer more.

That's why before we extracted a similar named utility, yet in other places did things like this:

```
				// Allocate bytes with enough size for all types.
 				expr := &ConstantExpression{Data: make([]byte, 8)}
```

Reply:
oh if you are referring to the impl.. I wanted to add another utility for only the lower bits, but found some things rely on reading 8 even if only 4 are needed. I didn't want to dig into why during the same change, but in any case that's why I did inline declaration vs initializing a byte slice and passing it to PutUint64 which does the same thing

Reply:
pretty sure there will be more maintenance, so if we want to change to two step I'll do it in the next hunt!